### ISSUE TITLE
Simplify the creation of a new project 

### ISSUE THREAD

**Issue by almet** [spiral-project/ihatemoney#525]
I like how Splittypie does this : https://splittypie.com/new.

It asks the user to select a name for the project, and in the same window, proposes to add the new participants.

**Comment by indatwood** [comment]
This screen seems nice. I'm attaching a screenshot so that everybody can see.

![image](https://user-images.githubusercontent.com/59884378/79993097-a971ab00-84b4-11ea-95c1-d71adc33221c.png)

Marking this as a good first bug, and pinging @eMerzh as he seems to have good usability insights.

**Comment by eMerzh** [comment]
didn't new splittypie, not really a huge fan... like participants are required to go forwardb but not marked as required, why choosing currency is SOOOO usefull at this stage.

And all in all our form ask for less informations ... 
I think the issue come down our "prefered" login method (the project+password) instead of the token.

My Ideal would be :

Screen 1:
Ask from project name + email

Screen 2 (already in the project):
Propose to invite people and (in the same action) add them as participants

**Comment by Glandos** [comment]
I agree with @eMerzh. The UI is nice and polished, but the UX is different. We all agreed that a currency should be optional, and participants are not mandatory for creation.

Adding participants is a one-shot task, so there's no need to have a "batch add" UI.

Some settings (like currency) could be available from the sidebar as a quick way to review and set them.

**Comment by Sammed98** [comment]
So, is anyone working on this? In case not, then could someone guide on whether we are proceeding with this issue and what UI changes are needed to be done.

**Comment by Glandos** [comment]
This task is very subjective and opinionated. Sometimes, we will disagree. The best advice is to start some rough sketching, and post it here, to serve as a base discussion.
The goal is to be sure that the bare minimal is required to create a project:
- Project name
- Project maintainer email

The code is even optional and could be generated by the server. Inviting other people can be done in a second step.
